app/src/main/java/com/peacecorps/pcsa/safety_tools/SafetyPlanBasicsContentFragment.java

```diff
@@ -41,11 +41,18 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         String content = getArguments().getString(SafetyPlanBasicsFragment.CONTENT_KEY);
         contenttoDisplay.setText(Html.fromHtml(content));
         contenttoDisplay.setMovementMethod(new ScrollingMovementMethod());
-        titleToDisplay.setText(Html.fromHtml(title));
-        titleToDisplay.setTypeface(null,Typeface.BOLD);
-        titleToDisplay.setTextSize(TypedValue.COMPLEX_UNIT_SP, 18);
-        titleToDisplay.setGravity(Gravity.CENTER);
-        getDialog().setTitle(title);
+        if(title != null) {
```

Comment:
In future, if you resuse this, move to a separate method. At this stage  this is okay.